cmake/Platform/Linux/Packaging/snapcraft.yaml.in

```diff
@@ -0,0 +1,64 @@
+name: o3de
+version: '${O3DE_INSTALL_VERSION_STRING}'
+summary: O3DE Engine
+description: |
+  O3DE Engine
```

Comment:
Think we need @o3de/sig-release to weigh in on what this should be? This will ultimately be put on the store page (though we can hand edit it, I think)

Reply:
Yes, waiting on text here, will update this once we get it.

Reply:
do we need a description field in engine.json?

Reply:
This should be a fairly descriptive string. Not sure if `engine.json` would support this, but see the one for CMake for an example: https://github.com/Crascit/cmake-snap/blob/master/snap/snapcraft.yaml

Reply:
Though in reading the CMake example, we should probably put in the license and an icon